i18n/zh/messages.zh.xlf

```diff
@@ -2162,15 +2162,15 @@
       </trans-unit>
       <trans-unit id="b0e6fb912650bbbe88ff81601b2dea87727abb6f" datatype="html">
         <source>Persistent Volume Claims</source>
-        <target state="new">持久化 Volume 请求</target>
+        <target state="new">持久化卷请求</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/resourcelist/persistentvolumeclaim/template.html</context>
           <context context-type="linenumber">21</context>
         </context-group>
       </trans-unit>
       <trans-unit id="796e42bec8a6996607cf1f3e80235b86d695804a" datatype="html">
         <source>Volume</source>
-        <target state="new">Volume</target>
+        <target state="new">卷</target>
```

Comment:
[Terms](https://github.com/k8smeetup/k8s-official-translation/blob/master/trans-glossary.md) should not be translated